lib/static/components/state/state-error.jsx

```diff
@@ -6,13 +6,23 @@ import {bindActionCreators} from 'redux';
 import PropTypes from 'prop-types';
 import {isEmpty, map, isFunction} from 'lodash';
 import ReactHtmlParser from 'react-html-parser';
+import escapeHtml from "escape-html";
+import ansiHtml from "ansi-html-community";
 import * as actions from '../../modules/actions';
 import ResizedScreenshot from './screenshot/resized';
 import ErrorDetails from './error-details';
 import Details from '../details';
 import {ERROR_TITLE_TEXT_LENGTH} from '../../../constants/errors';
 import {isAssertViewError, isImageDiffError, isNoRefImageError} from '../../../common-utils';
 
+ansiHtml.setColors({
+    reset: ["#", "#"],
+    cyan: "ff6188",
+    yellow: "5cb008",
+    magenta: "8e81cd",
+    green: "aa8720",
```

Comment:
Those are not cyan, yellow, magenta and green colors.
Just picked some with more contrast